worker/transcribee_worker/config.py

```diff
@@ -24,6 +24,8 @@ class Settings(BaseSettings):
         },
     }
 
+    KEEPALIVE_INTERVAL: float = 0.5
```

Comment:
Not that important, but this seems like a very short interval? 
I would have thought we might use something on the order of 5 seconds.

Reply:
Many tasks provide much finer progress updates and showing this in the UI quickly make the progress feel faster (at least that's what I found during development). Additionally we can change this using the env variable for every worker